packages/connect/jest.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    preset: '../../jest.config.base.js',
+    // REF-TODO: should be node (default?) testEnvironment: 'node',
```

Comment:
agree, it makes more sense use `jsdom` as default since most of our libraries are browser env based

Reply:
I'd leave this for further discussion later. We should also update jest to the latest version and this is a separate task.